civis_jupyter_notebooks/platform_persistence.py

```diff
@@ -105,10 +106,19 @@ def get_client():
 
 def setup_logging():
```

Comment:
Yeah. I spent a ton of time hacking. Jupyter uses ZMQ to forward stderr and stdout around, I think. For whatever reason, when IPython is starting up, the logging from running `civis_client_config.py` does not appear in the Jupyter server logs. I finally got it to appear by pointing the logger at `sys.__stderr__`. This is a special value that records the `stderr` stream when python starts, not the current value at `sys.stderr` (which can be changed by python code! Fun!). I needed to prevent the logger from sending messages up to its parents in order to eliminate duplicated messages too. I should prob retest this last thing. Finally, I adjusted the logging format to make it pretty.